### PR TITLE
Fix datetime when passed as a param

### LINKED ISSUE #191: Passing datetime as request body throws JSON serializable error.
Relates to #80. I am not sure it got resolved.

The flow here deals only for swagger_to_py flow and not the other way round. If there is a date-time param in request body, it is not currently transformed to a str to make `json.dumps` work.

`SwaggerTypeCheck` currently is being used both for to_wire and from_wire flow. This needs to be distinguished.

### PR DESCRIPTION
**Pull request by prat0318**
* docstrings and tests are yet to be added.

It checks if it is a `to_wire` flow, in that case, converts `datetime` to `str`.

Fixes #191

### CODE REVIEW

File: swaggerpy/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-version = '0.7.10'
```
Comment (analogue) [inline swaggerpy/__init__.py]: What happened to 0.7.11?

Comment (prat0318) [inline swaggerpy/__init__.py]: oh i think 0.7.11 was not released. will merge 12 with 11 changes.

**Comment by analogue** [comment]
:+1:

**Comment by prat0318** [comment]
Ack. Waiting for travis to be green.

**Pull request merged**